types/tailwindcss/resolveConfig.d.ts

```diff
@@ -0,0 +1,5 @@
+import { TailwindConfig } from './tailwind-config';
```

Comment:
there is no './tailwind-config.js` in the source, why this cannot be `lib/util/resolveConfig` import?

Reply:
It can't be `lib/util/resolveConfig` because it has to be:

```ts
import resolveConfig from 'tailwindcss/resolveConfig'
```

Is this what you mean?

Source:

https://tailwindcss.com/docs/configuration#referencing-in-java-script

Reply:
This is how to consume the package IMO, not how it's written:
https://github.com/tailwindlabs/tailwindcss/blob/master/resolveConfig.js#L4
https://unpkg.com/tailwindcss@2.0.2/resolveConfig.js
this methods returns outcome of calling the `resolvveConfigObjects`:
```ts
const resolveConfigObjects = require('./lib/util/resolveConfig').default
```
you can of course skip 'lib/util/resolveConfig' details and just write shared types somewhere. But I'd not create some artificial file for those, just put them into 'index.d.ts' as shared interfaces (types).